Source/WebCore/html/HTMLSummaryElement.cpp

```diff
@@ -98,10 +100,30 @@ bool HTMLSummaryElement::isActiveSummary() const
 
 static bool isClickableControl(EventTarget* target)
```

Comment:
It would be nice to get a link to the spec defining this function

Reply:
```c++
static bool isClickableControl(EventTarget* target)
```
> It would be nice to get a link to the spec defining this function

The spec doesn’t actually define “clickable control”. What it does define is “interactive content”, at https://html.spec.whatwg.org/multipage/dom.html#interactive-content.

Should I maybe rename the function to `isInteractiveContent(…)`?

Reply:
`isInteractiveContentWithinSummary` would work.

Reply:
> `isInteractiveContentWithinSummary` would work.

I went with `isInInteractiveContentWithinSummary` — since the target might not itself be interactive content, but just needs to be a descendant of interactive content.

If you prefer making it simply `isInteractiveContentWithinSummary` though, I’m happy to change it to that.